AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyAdminForTaskSignupStatusChangeHandler.cs

```diff
@@ -43,7 +43,7 @@ public NotifyAdminForTaskSignupStatusChangeHandler(AllReadyContext context, IMed
             {
                 var link = $"View event: http://{_options.Value.SiteBaseUrl}/Admin/Task/Details/{taskSignup.Task.Id}";
 
-                var subject = !string.IsNullOrEmpty(volunteer.FirstName) && !string.IsNullOrEmpty(volunteer.LastName)
+                var subject = volunteer.FirstName != null && volunteer.LastName != null

```

Comment:
@ExoltenOne, what I'm trying to say is we don't need to check for null. It's impossible for FirstName or LastName to be null b/c of how users are created in the system. This line should be deleted and this is how the subject should get assigned to the Subject property on the NotifyVolunteersViewModel:

```
var command = new NotifyVolunteersCommand
{
    ViewModel = new NotifyVolunteersViewModel
    {
        EmailMessage = message,
        HtmlMessage = message,
        EmailRecipients = new List<string> { adminEmail },
        Subject = $"{volunteer.FirstName} {volunteer.LastName}" 
    }
};
```